packages/angular/cli/lib/init.ts

```diff
@@ -16,6 +16,9 @@ import { SemVer } from 'semver';
 import { Duplex } from 'stream';
 import { isWarningEnabled } from '../utilities/config';
 
+// from @types/v8-profiler (no dependency to v8-profiler as we use v8-profiler-node8)
+import { Profiler } from 'v8-profiler'; // tslint:disable-line:no-implicit-dependencies
```

Comment:
Sorry, this should not be there. We don't want to depend v8-profiler, just use it if it's asked for. Maybe a dynamic import would be better?

Reply:
I think this only depends on @types/v8-profiler, but I may be wrong. I struggled a little with how to respect both no-any and no-implicit-any here. Should I simply add a disable on no-any or define a simple interface for that ?

Reply:
Replaced by a minimal Profiler type inspired by @types/v8-profiler.